controllers/reconciler.go

```diff
@@ -144,16 +172,22 @@ func (r *SingleClusterReconciler) Reconcile() (ctrl.Result, error) {
 		return reconcile.Result{}, e
 	}
 
-	if err := deployment.InfoQuiesceUndo(
+	if err = deployment.InfoQuiesceUndo(
 		r.Log,
 		r.getClientPolicy(), allHostConns,
 	); err != nil {
 		r.Log.Error(err, "Failed to check for Quiesced nodes")
 		return reconcile.Result{}, err
 	}
 
+	podList, err := r.getClusterPodList()
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
```

Comment:
Can we move this inside `validateAndReconcileAccessControl` like if `securityEnabledPods` is nil then get cluster pods list otherwise use the given list?
This way we don't have to get pod list in case of security disabled cluster

Reply:
we can do that, but passing nil as `securityEnabledPods` argument kind of looks like there is no security enabled pods in cluster even if all pods are security enabled. Just for the readability I did this way.